src/components/events/YearlyEvents.svelte

```diff
@@ -19,17 +21,29 @@
 	</p>
 	<div class="mt-3 flex flex-row">
 		<div class="flex items-center">
-			<input class="h-7 w-7" type="checkbox" id="for-all" bind:checked={for_all} />
+			<input
+				class="h-7 w-7"
+				type="checkbox"
+				id="for-all"
+				on:click={toggleForAll}
+				checked={filter_for_all}
+			/>
 			<label class="ml-4" for="for-all">Für Alle</label>
 		</div>
 		<div class="ml-10 flex items-center lg:ml-28">
-			<input class="h-7 w-7" type="checkbox" id="only-members" bind:checked={only_members} />
+			<input
+				class="h-7 w-7"
+				type="checkbox"
+				id="only-members"
+				on:click={toggleForAll}
+				checked={!filter_for_all}
+			/>
 			<label class="ml-4" for="only-members">Nur Mitglieder</label>
 		</div>
 	</div>
 	<ul class="flex flex-col lg:mt-12 lg:grid lg:grid-cols-2 lg:gap-10">
 		{#each events as event}
-			{#if (event.for_all && for_all) || (!event.for_all && only_members)}
+			{#if (event.for_all && filter_for_all) || (!event.for_all && !filter_for_all)}
```

Comment:
I think it would be nicer to filter the events in the `<script />` tag and make it reactive, so something like

```ts
$: filteredEvents = events.filter(e => event.for_all === filter_for_all)
```

and then here just iterate with `each` over `filteredEvents`

That way we do less logic in the template part of the component.